test/fixer/sqs_client_test.rb

```diff
@@ -0,0 +1,57 @@
+# encoding: utf-8
+
+require 'minitest_helper'
+
+describe Fixer::SqsClient do
+
+  let(:client) { Fixer::SqsClient.new }
+
+  let(:job_params) do
+    {
+      job: {
+        job_type: 'test',
```

Comment:
I hear ya @chrisrhoden 
FWIW - I was unconvinced by that article @farski sent, and still prefer single quote when there is no interpolation.